src/styles/blocks/catalog.scss

```
@@ -17,6 +17,14 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    
+    transition: transform 300ms ease-in-out,
+      color 300ms ease-in-out ;
+
+    &:hover {
+      transform: scale(1.2);
+      color: #34568b;
```

don't remember to create variables for colors
```suggestion
      color: $c-someColor;
```